frontend/components/blocks/international-campers-section.module.css

```
@@ -0,0 +1,78 @@
+  }
+
+  .globeContainer canvas {
+    touch-action: pan-y;
+  }
+}
+
+/* Phone: globe bleeds into gutters for a larger visual. */
+@media (max-width: 639px) {
+  .globeContainer {
+    --gw: min(620px, 100vw);
+    margin: 0 calc(-1 * var(--gutter, 24px));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)